src/main/java/com/ezasm/gui/EditorPane.java

```diff
@@ -184,6 +184,6 @@ public void updateHighlight(int line) {
     }
 
     public void resetHighlighter() {
-        highlighter = new LineHighlighter(HIGHLIGHT_COLOR, textArea);
+        highlighter = new LineHighlighter(Window.currentTheme().getCurrentLine(), textArea);
```

Comment:
getCurrentLine() is for the color of the line currently edited to be highlighted. You were probably on the right track with using some sort of yellow to be honest. My bad, getCurrentTheme is an unintuitive method name and should have had some javadoc attached to it.

Reply:
ahh got it

Reply:
Should I add a color for line highlighting? I dont see one currently

Reply:
You could just use the yellow provided.